crates/uv-tool/src/tool.rs

```diff
@@ -30,6 +32,7 @@ struct ToolWire {
     requirements: Vec<RequirementWire>,
     python: Option<String>,
     entrypoints: Vec<ToolEntrypoint>,
+    manpages: Option<Vec<ToolManpage>>,
```

Comment:
The manpages table is currently written only when necessary. We could also always include it if that's deemed more appropriate.